.docker/Dockerfile-template.ckan

```diff
@@ -0,0 +1,37 @@
+FROM openknowledge/ckan-dev:@CKAN_VERSION@
```

Comment:
could we try adding to github actions the container caching plugin (like m2/node cache plugin) this may make it even faster if we are not always getting the ckan-dev tag versions freshly.

https://github.com/marketplace/actions/docker-layer-caching
or
https://github.com/marketplace/actions/cache-container-images-action

Reply:
Possibly, but most of the time on this repo's builds is in testing, so there's probably more potential there, unless it's trivial to implement.

Reply:
Looks like https://github.com/marketplace/actions/docker-layer-caching is designed for caching the container that the workflow is executing in. Which isn't what we want; we instead need to cache one of the containers created by Docker Compose.

And https://github.com/marketplace/actions/cache-container-images-action doesn't support Docker.